.goreleaser.yaml

```diff
@@ -97,160 +97,3 @@ brews:
     homepage: *website
     description: *description
     license: "Apache License 2.0"
```

Comment:
nit: we should probably remove the `DOCKER_CLI_EXPERIMENTAL` var since it's no longer used